integrationtests/python_src/px4_it/mavros/tests.md

```diff
@@ -0,0 +1,16 @@
+
+
+
+
+
+
+| Test Name  | Description | Image |
+| ------------- | ------------- | ------------- |
+| [multirotor_box.mission](multirotor_box.mission "multirotor box") | Simple multirotor box mission. [Example](https://logs.px4.io/plot_app?log=c950b7f7-86cb-4b89-8674-b0f46265b823) | ![multirotor_box](images/multirotor_box.png) |
+| Offboard Attitude | Offboard attitude test with ROS. [Example](https://logs.px4.io/plot_app?log=17a5dbd3-c1a8-4d2e-97e5-bf81de048baf) | ![offboard_attitude](images/mc_offboard_attitude.png) |
```

Comment:
No link for offboard attitude/position tests?

Reply:
Those are separate little python (rospy) tests. https://github.com/PX4/Firmware/blob/master/integrationtests/python_src/px4_it/mavros/mavros_offboard_attctl_test.py

Reply:
Consider simplest format for markdown table - omit the outer pipes, use just three dashes to mark columns. IMO makes things more readable. 

```
Test Name  | Description | Image 
 --- | --- | ---
[multirotor_box.mission](multirotor_box.mission "multirotor box") | Simple multirotor box mission. [Example](https://logs.px4.io/plot_app?log=c950b7f7-86cb-4b89-8674-b0f46265b823) | ![multirotor_box](images/multirotor_box.png)
Offboard Attitude | Offboard attitude test with ROS. [Example](https://logs.px4.io/plot_app?log=17a5dbd3-c1a8-4d2e-97e5-bf81de048baf) | ![offboard_attitude](images/mc_offboard_attitude.png)

Reply:
To link to individual tests you will need to add an id for each test. You can do that with html spans. `<span id="example_whatever"></span>`

Reply:
Links aren't need for these tests, but we should probably say what they are doing... 

That leads to an issue with all the tests. e.g. new_1 and new_2 are the essentially the same, but new_2 has a WP in the middle and the land location is different. If we have new_2, why do we need new_1? Does the landing location mean anything that makes it different from the other that requires a separate test. Another example is: old_1 and old_2 contain the same commands with different locations. It is commented in these what the locations are -- what it is supposed to do. 

This is what we need in the description column. Of course, before this, we need to decide what we want to test and create more tests. But, I'd forget about these initially and first come up with descriptions for tests we think we need. Then we can look at the tests we have, see if they fit the bill, or make new ones -- using QGC .plan format.

Reply:
Yes, the current ones need to be tossed. I just want to get all the pieces in place (test definition, jenkins portions, basic description) so we can bring up a useful level of testing. I also think it's very important for what Jenkins posts to flight review to either link to this documentation or include it directly.

Reply:
I think a link is good idea to quickly see what it should be doing and look like on the map. Putting it in the feedback section should work fine.